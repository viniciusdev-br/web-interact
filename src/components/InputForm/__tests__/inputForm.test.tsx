import renderer from 'react-test-renderer';
import { describe, it, expect } from "vitest";
import { InputForm } from '..';
import { toJson } from '../../../../tests/utils';


describe('InputForm', () => {
    it('Snapshot default do componente de input de formulário', () => {
        const component = renderer.create(<InputForm placeholder='Nome do usuário'/>);
        let tree = toJson(component);
        expect(tree).toMatchSnapshot();
    });
});