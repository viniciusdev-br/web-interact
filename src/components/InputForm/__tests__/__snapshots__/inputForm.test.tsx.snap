// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`InputForm > Snapshot default do componente de input de formulário 1`] = `
<input
  className="input-form"
  placeholder="Nome do usuário"
/>
`;
